usersession/agent/rest_api.go

```diff
@@ -38,6 +47,11 @@ var (
 		Path: "/v1/session-info",
 		GET:  sessionInfo,
 	}
+
+	servicesCmd = &Command{
+		Path: "/v1/services",
```

Comment:
/services makes one think that is high-level and a deals in snap services, instead of systemd units.  Maybe we should be consistent with other naming here and call this /service-control

Reply:
Good idea.  I've moed it to `/v1/service-control` now.